rke/templates/coredns.go

```diff
@@ -97,11 +97,15 @@ metadata:
     addonmanager.kubernetes.io/mode: Reconcile
     kubernetes.io/name: "CoreDNS"
 spec:
-  replicas: 1
   strategy:
+{{if .UpdateStrategy}}
+{{ toYaml .UpdateStrategy | indent 4}}
+{{else}}
     type: RollingUpdate
     rollingUpdate:
-      maxUnavailable: 1
```

Comment:
I think we should be passing our defaults from RKE rather than trying to define defaults in the template here